packages/snap-preact/components/src/components/Templates/Recommendation/Recommendation.tsx

```diff
@@ -165,10 +172,8 @@ export const Recommendation = observer((properties: RecommendationProps): JSX.El
 							hideButtons={hideButtons}
 							loop={loop}
 							pagination={pagination}
-							breakpoints={breakpoints}
 							{...subProps.carousel}
 							{...additionalProps}
-							{...displaySettings}
 						>
```

Comment:
Do you think pulling either of these out will cause issues with backwards compatibility? My thinking was that all of the props used in the current breakpoint would be spread into additionalProps anyways so it would still work. But curious your thoughts